index.html

```diff
@@ -425,7 +425,7 @@ <h5 lang="en">Change language</h5>
         </div>
         <div class="clearfix"></div>
       </div>
-      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved.</span></p>
+      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved. </span><span lang="de"><a href="https://de.opensuse.org/Impressum">Impressum</a>.</span></p>
```

Comment:
the lang attribute marks the string or translation. So in the current form translators would translate both "Impressum" as well as where the link points at. What's correct depends on what is needed really. Do you want the "Impressum" to be only displayed when the language is set to German, or do you want to also have an "Imprint" link that is always shown in any language?